src/main/java/mate/academy/springboot/datajpa/model/Category.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
```

Comment:
is it a good practice to mark our entity with Data annotation? Maybe will be better to use Getter
Setter only